include/cantera/kinetics/Kinetics.h

```diff
@@ -597,42 +597,121 @@ class Kinetics
     //! @addtogroup derivGroup
     //! @{
 
-    //! @name Routines to Calculate Kinetics Derivatives (Jacobians)
-    //!
-    //! Kinetics derivatives are calculated with respect to temperature, pressure,
-    //! molar concentrations and species mole fractions for forward/reverse/net rates
-    //! of progress as well as creation/destruction and net production of species.
-    //!
-    //! The following suffixes are used to indicate derivatives:
-    //!  - `_ddT`: derivative with respect to temperature (a vector)
-    //!  - `_ddP`: derivative with respect to pressure (a vector)
-    //!  - `_ddC`: derivative with respect to molar concentration (a vector)
-    //!  - `_ddX`: derivative with respect to species mole fractions (a matrix)
-    //!  - `_ddCi`: derivative with respect to species concentrations (a matrix)
-    //!
-    //! Settings for derivative evaluation are set by keyword/value pairs using
-    //! the methods getDerivativeSettings() and setDerivativeSettings().
-    //!
-    //! For BulkKinetics, the following keyword/value pairs are supported:
-    //!  - `skip-third-bodies` (boolean) ... if `false` (default), third body
-    //!    concentrations are considered for the evaluation of jacobians
-    //!  - `skip-falloff` (boolean) ... if `false` (default), third-body effects
-    //!    on rate constants are considered for the evaluation of derivatives.
-    //!  - `rtol-delta` (double) ... relative tolerance used to perturb properties
-    //!    when calculating numerical derivatives. The default value is 1e-8.
-    //!
-    //! For InterfaceKinetics, the following keyword/value pairs are supported:
-    //!  - `skip-coverage-dependence` (boolean) ... if `false` (default), rate constant
-    //!    coverage dependence is not considered when evaluating derivatives.
-    //!  - `skip-electrochemistry` (boolean) ... if `false` (default), electrical charge
-    //!    is not considered in evaluating the derivatives and these reactions are
-    //!    treated as normal surface reactions.
-    //!  - `rtol-delta` (double) ... relative tolerance used to perturb properties
-    //!    when calculating numerical derivatives. The default value is 1e-8.
-    //!
-    //! @warning  The calculation of derivatives is an experimental part of the
-    //!      %Cantera API and may be changed or removed without notice.
-    //! @{
+    /**
+     * @anchor kinDerivs
+     * @par Routines to Calculate Kinetics Derivatives (Jacobians)
+     * @name
+     *
+     * Kinetics derivatives are calculated with respect to temperature, pressure,
```

Comment:
Interesting, what Doxygen version were you using? It apparently has something to do with adding this section to both "kinetics" and "derivatives" groups. However, I don't see the 'wall': With Doxygen=1.9.1, I get no italics (under derivatives):

![image](https://github.com/Cantera/cantera/assets/12603999/455dcb03-47cc-4345-983c-0bce41fbc99c)

and (under Kinetics), normal text, although list items are italicized.

![image](https://github.com/Cantera/cantera/assets/12603999/77a6ee4f-494a-4948-82ac-3067ea2886ad)

This is also the same behavior as you currently see under merged development documentation: https://cantera.org/documentation/dev/doxygen/html/d9/d08/group__derivGroup.html#func-members and 
https://cantera.org/documentation/dev/doxygen/html/d4/dc4/classCantera_1_1Kinetics.html#pub-methods

I thought that it may have something to do with adding the 'anchor', but it makes little difference. I *believe* that this may be a CSS issue?

Reply:
I'm seeing this with Doxygen 1.9.6, on the page for the `Kinetics` class:
<img width="978" alt="image" src="https://github.com/Cantera/cantera/assets/644977/8f09c796-3e4b-49ec-ac80-6153ec98c81a">

I had not noticed that this content was also being duplicated into the `modules` section. Looking now, I see it is appearing on that page correctly, without the unnecessary italicization.

Reply:
In any case, I think the actual content here is a good addition, and I'm not that concerned about the formatting details.